aactivator.py

```diff
@@ -246,23 +246,38 @@ def security_check(path):
         )
 
 
+def commands(commands, always=None):
+    """
+    run a set of commands, stopping on the first error
```

Comment:
this docstring seems false, this function does not "run a set of commands, stopping on the first error"